client/components/dialog/Dialog.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import Dialog from './Dialog';
+
+describe('<Dialog />', () => {
+  describe('Render', () => {
+    describe('When options are empty, must return an empty div', () => {
```

Comment:
Fica melhor de ler quando os testes você coloca a condição no texto do describe o que você espera no it. Por exemplo:

```js
describe('when options are empty', () => {
   it('returns an empty div', () => {
      ...
   })
})
`´`

Reply:
resolvido